src/components/AddBook.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { bookTitle, bookAuthor, setBookTitle, setBookAuthor } from '../redux/books/books'; //eslint-disable-line
 
 const AddBook = () => { //eslint-disable-line
```

Comment:
- [ ] **Required:** You have disabled linters in virtually all the project files. Linters are meant to enforce a community-style guide. Kindly fix them rather than disable them.